uaa/src/main/webapp/WEB-INF/spring-servlet.xml

```diff
@@ -339,6 +339,11 @@
     </bean>
 
 
+    <bean id="clientDetailsValidator" class="org.cloudfoundry.identity.uaa.client.ClientAdminEndpointsValidator">
```

Comment:
this has happened to me as well, where the beans are difficult to move. once XML is all gone, maybe we can go over the bean diagram and see if we can simplify it